JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -3,35 +3,57 @@
 //  Created by yagom. 
 //  Copyright © yagom academy. All rights reserved.
 //
+import Foundation
 
 class FruitStore {
-    private(set) var fruitsInventory: [Fruit: Int] = [:]
+    static let shared: FruitStore = FruitStore()
+    
+    private(set) var inventoryList: [Fruit: Int] = [:] {
+        didSet {
+            NotificationCenter.default.post(name: .inventoryChanged,
+                                            object: nil)
+        }
+    }
     let defaultValueOfInventory = 10
     
-    init() {
+    private init() {
         Fruit.allCases.forEach {
-            fruitsInventory[$0] = defaultValueOfInventory
+            inventoryList[$0] = defaultValueOfInventory
         }
     }
     
-    func increaseInventory(of fruit: Fruit, by amount: Int) {
+    func increase(of fruit: Fruit, by amount: Int) {
```

Comment:
by 보다는 amount 그 자체를 써주는게 사용처에서는 직관 적일 것 같아요 ㅎㅎ by를 사용하다보니 ~에 의해서 변경되는 것처럼 느껴져서요

혹은 amount를 변경한다는 것을 메서드 명에 녹여주고 뒤에서는 to를 써줘도 괜찮을 것 같아요 ㅎㅎ (이런느낌으로?)
```suggestion
    func increaseAmount(of fruit: Fruit, to amount: Int) {
```

Reply:
좋은 의견 감사합니다!

Reply:
해당 부분은 필요없는 함수라 제거했습니다..!
다음에 네이밍을 신경써서 작성하겠습니다!